### PR TITLE
Fixing build to copy correct dependencies

### PR DESCRIPTION
**Pull request by Evmaus-MS**
This fixes the build so that the native dependencies are properly copied.

### CODE REVIEW

File: src/BinSkim.Driver/BinSkim.Driver.csproj [hunk 1]
```diff
@@ -64,8 +64,8 @@
 
   <ItemGroup>
     <!-- May need to be changed to x86 on a 32 bit machine. -->
-    <MsDiaLibs Include="..\..\refs\x64\*" Condition="'$(Configuration)' == 'Debug' And '$(TargetFramework)'=='$(NetRuntimeVersion)' And '$(RuntimeIdentifier)'=='win7-x86'" />
```
Comment (michaelcfanning) [inline src/BinSkim.Driver/BinSkim.Driver.csproj]: >'$(Configuration)' == 'Debug' And '$(T [](start = 53, length = 38)

ouch! :)

File: src/BinSkim.Driver/BinSkim.cs [hunk 2]
```diff
@@ -10,6 +10,7 @@
 using System.Linq;
 using System.Runtime.InteropServices;
 using Microsoft.CodeAnalysis.Sarif;
+using Microsoft.CodeAnalysis.Sarif.Driver.Sdk;
```
Comment (michaelcfanning) [inline src/BinSkim.Driver/BinSkim.cs]: >CodeAnalysis.Sarif.Driver [](start = 16, length = 25)

is this change actually required? just curious

Comment (Evmaus-MS) [reply src/BinSkim.Driver/BinSkim.cs]: Yeah, the namespace for EntryPointUtilities in the driver was changed at some point.
